moveit_core/collision_detection_fcl/test/test_fcl_collision_detection.cpp

```diff
@@ -460,13 +460,13 @@ TEST_F(FclCollisionDetectionTester, TestCollisionMapAdditionSpeed)
     poses.push_back(Eigen::Isometry3d::Identity());
     shapes.push_back(shapes::ShapeConstPtr(new shapes::Box(.01, .01, .01)));
   }
-  ros::WallTime start = ros::WallTime::now();
+  auto start = std::chrono::system_clock::now();
   cworld_->getWorld()->addToObject("map", shapes, poses);
-  double t = (ros::WallTime::now() - start).toSec();
+  double t = (std::chrono::system_clock::now() - start).count();
   EXPECT_GE(1.0, t);
   // this is not really a failure; it is just that slow;
   // looking into doing collision checking with a voxel grid.
-  ROS_INFO_NAMED("collision_detection.fcl", "Adding boxes took %g", t);
+  // ROS_INFO_NAMED("collision_detection.fcl", "Adding boxes took %g", t);
```

Comment:
why is this disabled? seems unrelated to porting to ROS2

Reply:
https://github.com/ros-planning/moveit2/pull/41/commits/ed4f5127eb6daea4f845d7cf1adb44db54a54e1f